hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -112,6 +112,10 @@ public Response list(
     ContinueToken decodedToken =
         ContinueToken.decodeFromString(continueToken);
 
+    // Assign marker to startAfter. for the compatibility of aws api v1
+    if (startAfter == null && marker != null) {
```

Comment:
Below we are not setting marker should we also set that also, if the marker is not null?
 L136: response.setMarker("");

Reply:
Done.